woocommerce/woocommerce-ios
# Login Enhancement: Show Jetpack error after logging in with WP.com account

Closes: #7222


### Description

To improve the login experience, we want to move the Jetpack error in the login with site address flow after the login succeeds. This PR handles this improvement with 2 changes:
- Remove the check for Jetpack before presenting the email/password screens
- Check for the Jetpack requirement first thing in the login epilogue step

### Testing instructions

- Prerequisite: make sure that your account is associated with a self-hosted site that has WP and no Jetpack connection.
- Log out of your current account if necessary.
- On the login prologue screen, select Enter Your Store Address then enter the address of the site mentioned in the prerequisite.
- Enter the associated account's email and password.
- Notice that after the login succeeds, the Jetpack requirement screen is presented.

Feel free to test with an eligible site (has WP, Jetpack, WooCommerce) to see if it still works properly.

### Screenshots


https://user-images.githubusercontent.com/5533851/177519243-14ff7d76-2022-4217-809c-367a15c3b43c.mp4

(not recording the login flow for privacy reasons)

---
